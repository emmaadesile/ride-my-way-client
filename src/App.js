import React from "react";
import AppRoutes from "./AppRoutes";

const App = () => (
  <div>
    <AppRoutes />
  </div>
);

export default App;
