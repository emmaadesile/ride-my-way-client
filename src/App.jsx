import React from "react";
import AppRoutes from "./AppRoutes.jsx";

const App = () => (
  <div>
    <AppRoutes />
  </div>
);

export default App;
