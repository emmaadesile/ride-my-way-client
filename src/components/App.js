import React from 'react';
import { BrowserRouter as Router, Route } from 'react-router-dom';
import HomePage from './homepage/Homepage';
import LoginForm from './login/Login';
import SignupForm from './signup/Signup';

const App = () => {
  return (
    <Router>
      <div>
        <Route exact path='/' component={HomePage} />
        <Route path='/signup' component={SignupForm} />
        <Route path='/login' component={LoginForm} />
      </div>
    </Router>
  )
}

export default App;