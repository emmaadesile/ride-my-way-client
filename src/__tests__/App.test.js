import React from 'react';
import Enzyme, { shallow, mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';
import App from '../App';

Enzyme.configure({ adapter: new Adapter() });

describe("Tests for App component", () => {
  test("renders", () => {
    const app = shallow(<App />)
    // expect(app.JSON().toMatchSnapshot());
    expect(app.exists()).toBe(true);
  });
})