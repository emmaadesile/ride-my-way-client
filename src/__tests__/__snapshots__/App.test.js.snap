// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <div
    className="navbar"
  >
    <div
      className="container navbar__container"
    >
      <li
        className="logo"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <img
            alt="logo"
            src="rideLogo.png"
          />
        </a>
      </li>
      <ul>
        <li>
          <a
            href="/about"
            onClick={[Function]}
          >
            About
          </a>
        </li>
        <li>
          <a
            href="/auth/login"
            onClick={[Function]}
          >
            Log in
          </a>
        </li>
        <li>
          <a
            className="register"
            href="/auth/signup"
            onClick={[Function]}
          >
            Register
          </a>
        </li>
      </ul>
    </div>
  </div>
  <div
    className=""
  >
    <header
      className="header header__content"
    >
      <h1>
        Get the best deals on your rides
      </h1>
      <h2>
        Join ride my way community and save time
        <br />
        and money on your commute everyday
      </h2>
      <a
        href="/signup"
        onClick={[Function]}
      >
        Get Started
      </a>
    </header>
    <div
      className="intro text-center "
    >
      <h4
        className="lead"
      >
        Ride My Way In Your City
      </h4>
      <p
        className="lead "
      >
        Wherever you are in Nigeria, count on Ride My Way for rides in minutes!
        <br />
        From Lagos to Abuja, Ride My Way is only a few clicks away.
      </p>
    </div>
    <section
      className="features "
    >
      <h4
        className="feature-heading"
      >
        Why Ride With Us
      </h4>
      <div
        className="features-grid mt-1 "
      >
        <div
          className="feature feature-1 "
        >
          <div
            className="icon "
          >
            <img
              alt="confirmed"
              src="confirmed.png"
            />
          </div>
          <h5>
            Easy Booking
          </h5>
          <p>
            Select your preferred rides and ride with only those you wish to. Riding with your colleagues, friends, and family has never been easier.
          </p>
        </div>
        <div
          className="feature feature-2 "
        >
          <div
            className="icon "
          >
            <img
              alt="customer care"
              src="customerCare.png"
            />
          </div>
          <h5>
            24/7 Customer Care
          </h5>
          <p>
            Our agents are always online 24/7 to sort out any issues you might be experiencing. We put our customers first before anything else.
          </p>
        </div>
        <div
          className="feature feature-3 "
        >
          <div
            className="icon "
          >
            <img
              alt="time management"
              src="timeManagement.png"
            />
          </div>
          <h5>
            Ride on time
          </h5>
          <p>
            Dont waste any more time waiting for rides. With our pool of several cars and drivers in your area, get to your destination on time.
          </p>
        </div>
      </div>
    </section>
    <section
      className="about"
    >
      <div
        className="about__details"
      >
        <h4>
          About Us
        </h4>
        <p>
          As the most reliable carpooling service, we help thousands of users, both passengers and drivers make the most of their rides. Passengers have control over the rides they wish to join and drivers have control over rides they wish to accept. Our cars span several routes in your city and we guarantee that you will never be stranded.
        </p>
      </div>
    </section>
  </div>
  <footer
    className="footer "
  >
    <p>
      Â© 2018. Made by Emmanuel
    </p>
  </footer>
</div>
`;
